include/circt/Dialect/Ibis/IbisInterfaces.td

```diff
@@ -18,7 +18,38 @@ include "circt/Dialect/HW/HWOpInterfaces.td"
 include "mlir/IR/OpBase.td"
 include "mlir/IR/SymbolInterfaces.td"
 
-def PortOpInterface : OpInterface<"PortOpInterface"> {
+def NamedInnerSymbol : OpInterface<"NamedInnerSymbolInterface", [InnerSymbol]> {
+  let cppNamespace = "circt::ibis";
+  let description = [{
+    An interface for operations which define inner symbols with an optional
+    namehint.
+  }];
+
+  let methods = [
+    InterfaceMethod<
+      "Returns the namehint attribute of this op",
+      "mlir::StringAttr", "getNameHintAttr",
```

Comment:
Let's not call this a 'namehint'. Too easily confused with the sv.namehint attribute. Is there any reason why you don't just call it 'name'?

Reply:
There's a crazy amount of aliasing going on here in the `name`-related namespace. if one adds an attribute in ODS as `name`, that'll define `getName` functions, which the interface would then alias with... it feels like `getName` is almost _too_ generic to be a valid interface function name. Maybe `getSymbolName`? - ideas are welcome.